site/src/pages/TemplateVersionEditorPage/TemplateVersionEditor.tsx

```diff
@@ -575,62 +573,58 @@ export const TemplateVersionEditor: FC<TemplateVersionEditorProps> = ({
-                        borderBottom: `1px solid ${theme.palette.divider}`,
-                        borderLeft: `2px solid ${theme.palette.error.main}`,
-                      }}
-                    >
-                      <AlertTitle>Error during the build</AlertTitle>
-                      <AlertDetail>{templateVersion.job.error}</AlertDetail>
-                    </Alert>
-                  </div>
-                )}
-
-                {buildLogs && buildLogs.length === 0 && (
-                  <Loader css={{ height: "100%" }} />
-                )}
-
-                {buildLogs && buildLogs.length > 0 && (
-                  <WorkspaceBuildLogs
-                    css={styles.buildLogs}
-                    hideTimestamps
-                    logs={buildLogs}
-                  />
-                )}
-              </div>
-
-              <div
-                css={[
-                  {
-                    display: selectedTab !== "resources" ? "none" : undefined,
+              {selectedTab === "logs" && (
+                <div
+                  ref={buildLogsRef}
+                  css={{
+                    display: "flex",
                     height: selectedTab ? 280 : 0,
-                  },
-                  styles.resources,
-                ]}
-              >
-                {resources && (
-                  <TemplateResourcesTable
-                    resources={resources.filter(
-                      (r) => r.workspace_transition === "start",
-                    )}
-                  />
-                )}
-              </div>
+                    flexDirection: "column",
+                    overflowY: "auto",
+                  }}
+                >
+                  {templateVersion.job.error && (
+                    <div>
+                      <Alert
+                        severity="error"
+                        css={{
+                          borderRadius: 0,
+                          border: 0,
+                          borderBottom: `1px solid ${theme.palette.divider}`,
+                          borderLeft: `2px solid ${theme.palette.error.main}`,
+                        }}
+                      >
+                        <AlertTitle>Error during the build</AlertTitle>
+                        <AlertDetail>{templateVersion.job.error}</AlertDetail>
+                      </Alert>
+                    </div>
+                  )}
+
+                  {buildLogs && buildLogs.length > 0 ? (
+                    <WorkspaceBuildLogs
+                      css={styles.buildLogs}
+                      hideTimestamps
+                      logs={buildLogs}
+                    />
+                  ) : (
+                    <Loader css={{ height: "100%" }} />
+                  )}
+                </div>
+              )}
+
+              {selectedTab === "resources" && (
+                <div
+                  css={[{ height: selectedTab ? 280 : 0 }, styles.resources]}
```

Comment:
Just in case, could the order of the array elements be flipped?

My first thought when I saw this was that I'd have to go through the rest of the code to see if the height was being overridden by `styles.resources`. With the order flipped, it's clear that the height object will always win out

Reply:
Good catch, I can easily refactor this 🙏